### ISSUE TITLE
Request: stand-alone package for update logic/UI

### ISSUE THREAD

**Issue by nex3** [top-ghost/octobug#2]
I definitely want to use this for [my own site](https://github.com/nex3/nex3.github.io), but it's already all set up without using this template. I can copy in the relevant chunks of code, of course, but that's tough to keep up-to-date with upstream changes. Any chance the update-specific stuff could be pulled out?

**Comment by top-ghost** [comment]
yeah, we should definitely do this. I have an 11ty site for my "professional" presence as well and I'd love to be able to post more easily there. Good idea 👍
